src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
```

Comment:
its better to give self-explained name of variable for example MovieCardProps

Reply:
No need, conversely - It's better way